standards.md

```diff
@@ -1,11 +1,15 @@
 # Earth System Metrics and Diagnostics Standards
 
 ## Author list
+Authors are presented in order of last name:  
+
+Chih-Chieh Chen, National Center for Atmospheric Research  
 Dani Coleman, National Center for Atmospheric Research  
 Tom Jackson, NOAA Geophysical Fluid Dynamics Laboratory   
```

Comment:
Tom's affiliation is not NOAA GFDL any more. Not sure how to address this. I can contact him if that is the route

Reply:
Was Tom affiliated with NOAA GFDL at the time that he contributed to the standards? If so, it seems legitimate to leave that affiliation here. Similar to how journal articles would have author affiliations from the time of publication.

Reply:
He was with SAIC. I’ll let John comment on this further.